cartographer_rviz/cartographer_rviz/ogre_slice.cc

```diff
@@ -142,4 +144,8 @@ void OgreSlice::SetAlpha(const float alpha) {
   parameters->setNamedConstant("u_alpha", alpha);
 }
 
+void OgreSlice::SetVisibility(bool visible) {
```

Comment:
Visibility is a strongly overloaded term here. You already track the 'visibility' bit for each slice in DrawableSubmap. Maybe SetOgreNodeVisibility. Alternatively, you could move the 'visibility' bit into this class from DrawableSubmap and have a `UpdateNodeVisibility(bool submap_visibility)`.